src/gui/MovieImportDialog.cpp

```diff
@@ -109,56 +109,63 @@ void MyMovieImportDialog::AddFilesClick(wxCommandEvent& event) {
     wxString current_extension;
 
     if ( openFileDialog.ShowModal( ) == wxID_OK ) {
+
         wxArrayString selected_paths;
         wxArrayString final_paths;
         openFileDialog.GetPaths(selected_paths);
 
-        PathListCtrl->Freeze( );
```

Comment:
How come you are taking out the Freeze() Thaw()?

Reply:
@timothygrant80 

Please refer to my inline comment to answer this question.

Feel free to revert that bit. My primary goal is to maintain parallel functionality - it is okay if the implementation differs slightly.